src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+:root {
+  --outline: 0;
+  --border: 1px solid #e1e7ed;
+  --border-radius: 4px;
+  --main-text-color: #3d4e61;
+  --font-family: "Avenir", sans-serif;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.container {
+  margin-top: 20px;
+}
+
+.form {
+  position: relative;
+  height: 70px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px #3d4e6133;
+  border-radius: var(--border-radius);
+}
+
+.image {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+  cursor: text;
+}
+
+.search {
+  outline: var(--outline);
+  padding-left: 62px;
+  border: var(--border);
+  border-radius: var(--border-radius);
+  font-size: 16px;
+  font-family: var(--font-family);
+  line-height: 22px;
+  color: var(--main-text-color);
+}
+
+.style {
+  width: 100%;
+  height: 100%;
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: var(--border);
+  border-radius: 3px;
+  text-shadow: 0 4px 4px rgba(0,0,0,0.25);
+  font-weight: 800;
+}
+
+.search:focus::placeholder {
+  text-shadow: 0 4px 4px rgba(0,0,0,0.25);
+  font-weight: 800;
+}
+
+.form--small {
+  position: relative;
+  margin-top: 20px;
+  height: 42px;
+}
+
+.img--small {
+  position: absolute;
```

Comment:
Consider using a more descriptive class name for small image. For example, `.search-small-image`. 🤔